### PR TITLE
Updated project hub links to leonardo [MKC-820]

### PR DESCRIPTION
**Pull request by jhansson-ard**
## What This PR Changes
- Updated the links to project hub for the leonardo projects

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Comment by jhansson-ard** [comment]
@karlsoderby please take a look when you can :)

File: content/retired/06.getting-started-guides/ArduinoLeonardoMicro/ArduinoLeonardoMicro.md [hunk 1]
```diff
@@ -65,9 +65,9 @@ After the compilation and upload process, you should see the message _Done Uploa
...
+Now that you have set up and programmed your Leonardo, Leonardo ETH or Micro board, you may find inspiration in our [Project Hub](https://projecthub.arduino.cc/search?value=arduino+leonardo) tutorial platform.
 
-<iframe frameborder='0' height='410' scrolling='no' src='https://create.arduino.cc/projecthub/onyx/gesture-mouse-a4320a/embed?use_route=project' width='354' style='margin-top:30px; margin-right:20px'></iframe><iframe frameborder='0' height='410' scrolling='no' src='https://create.arduino.cc/projecthub/windows-iot/coffeeauth-grinder-authentication-system-d7cb38/embed?use_route=project' width='354' style='margin-top:30px'></iframe>
+<iframe frameborder='0' height='410' scrolling='no' src='https://www.hackster.io/onyx/gesture-mouse-a4320a' width='354' style='margin-top:30px; margin-right:20px'></iframe><iframe frameborder='0' height='410' scrolling='no' src='https://create.arduino.cc/projecthub/windows-iot/coffeeauth-grinder-authentication-system-d7cb38/embed?use_route=project' width='354' style='margin-top:30px'></iframe>
```
Comment (jhansson-ard) [inline content/retired/06.getting-started-guides/ArduinoLeonardoMicro/ArduinoLeonardoMicro.md:70]:
```suggestion
<iframe frameborder='0' height='410' scrolling='no' src='https://www.hackster.io/onyx/gesture-mouse-a4320a' width='354' style='margin-top:30px; margin-right:20px'></iframe><iframe frameborder='0' height='410' scrolling='no' src='https://www.hackster.io/windows-iot/coffeeauth-grinder-authentication-system-d7cb38' width='354' style='margin-top:30px'></iframe>

```

File: content/retired/06.getting-started-guides/ArduinoLeonardoMicro/ArduinoLeonardoMicro.md [hunk 2]
```diff
@@ -65,9 +65,10 @@ After the compilation and upload process, you should see the message _Done Uploa
 
 ### Tutorials
 
-Now that you have set up and programmed your Leonardo, Leonardo ETH or Micro board, you may find inspiration in our [Project Hub](https://create.arduino.cc/projecthub/products/arduino-leonardo) tutorial platform.
+Now that you have set up and programmed your Leonardo, Leonardo ETH or Micro board, you may find inspiration in our [Project Hub](https://projecthub.arduino.cc/search?value=arduino+leonardo) tutorial platform.
+
+<iframe frameborder='0' height='410' scrolling='no' src='https://www.hackster.io/onyx/gesture-mouse-a4320a' width='354' style='margin-top:30px; margin-right:20px'></iframe><iframe frameborder='0' height='410' scrolling='no' src='https://www.hackster.io/windows-iot/coffeeauth-grinder-authentication-system-d7cb38' width='354' style='margin-top:30px'></iframe>
```
Comment (jhansson-ard) [inline content/retired/06.getting-started-guides/ArduinoLeonardoMicro/ArduinoLeonardoMicro.md:70]:
```suggestion
* [Gesture Mouse](https://www.hackster.io/onyx/gesture-mouse-a4320a)
* [Grinder Authentication System](https://www.hackster.io/windows-iot/coffeeauth-grinder-authentication-system-d7cb38)
```

**Review by jacobhylen — APPROVED** [review]
LGTM

**Pull request merged**